crontab/import_pg_catalog.php

```diff
@@ -3,6 +3,7 @@
 include_once($relPath.'base.inc');
 include_once($relPath.'misc.inc');
 include_once($relPath.'Stopwatch.inc');
+include_once($relPath.'pg.inc');
 
 require_localhost_request(true /* deny_cli */);
```

Comment:
I feel comfortable with your changes to this file as-is, but if you _wanted_ to test it, you could do so by commenting this out and accessing the page via your web browser.

Reply:
Thanks -- yes, I like being able to test my code changes. 🙂